frontend/src/app/shared/components/file-links/storage-file-list-item/storage-file-list-item.html

```diff
@@ -0,0 +1,29 @@
+<label
+  class="spot-list--item-action"
+>
+  <spot-checkbox
+    [checked]="selected"
+    (click)="changeSelection.emit()"
+  ></spot-checkbox>
+  <span
+    class="spot-icon spot-icon_{{fileLinkIcon.icon}} op-files-tab--icon op-files-tab--icon_{{fileLinkIcon.clazz}}"
+  ></span>
+
+  <span
+    [textContent]="storageFile.name"
+    [title]="storageFile.name"
+    class="spot-list--item-title op-files-tab--file-list-item-title"
+  ></span>
+
+  <span
+    *ngIf="showDetails"
+    class="op-files-tab--file-list-item-text"
+    [textContent]="infoTimestampText"
+  ></span>
+
+  <div
+    #avatar
+    *ngIf="showDetails"
+    class="op-files-tab--file-list-item-avatar"
+  ></div>
```

Comment:
Can we do this with an `<op-principal></op-principal>`? Saves you from having to do a manual render in `ngAfterViewInit`

Reply:
this didn't work in the original file list, hence I didn't even tried to replace it here. I can have another look, though, about the why it didn't work.